src/PointwiseFunctions/AnalyticData/Xcts/BinaryWithGravitationalWaves.cpp

```diff
@@ -166,18 +166,19 @@ void BinaryWithGravitationalWavesVariables<DataType>::operator()(
     }
   }
 
+  const auto auxiliar_one = tenex::evaluate<ti::I, ti::J>(
+      inv_conformal_metric(ti::I, ti::K) * inv_conformal_metric(ti::J, ti::L) *
+      dt_conformal_metric(ti::k, ti::l));
+  const auto auxiliar_two = tenex::evaluate<ti::I, ti::J>(
+      inv_conformal_metric(ti::I, ti::K) * inv_conformal_metric(ti::J, ti::L) *
+      conformal_metric(ti::k, ti::l));
```

Comment:
This is unnecessary. The result is the inverse conformal metric, which you already have!